src/cover-hero.tsx

```diff
@@ -0,0 +1,191 @@
+                  alternative: vars.colors.backgroundAlternative,
+              }[variant ?? 'default'];
+
+        const textShadow = hasMedia ? '0 0 15px rgba(0, 0, 0, 0.4)' : undefined;
+
+        const mainContent = (
+            <div className={styles.mainContent}>
+                {headline && <Box paddingBottom={{desktop: 8, tablet: 8, mobile: 16}}>{headline}</Box>}
+                <Stack space={16}>
+                    <Stack space={8}>
+                        {pretitle && (
+                            <div className={styles.sixColumns}>
+                                <Text3 regular truncate={pretitleLinesMax} textShadow={textShadow}>
+                                    {pretitle}
+                                </Text3>
+                            </div>
+                        )}
+                        <Text8 as={titleAs} truncate={titleLinesMax} textShadow={textShadow}>
+                            {title}
+                        </Text8>
+                    </Stack>
+                    {description && (
+                        <div className={styles.sixColumns}>
+                            <Text3
+                                as="p"
+                                regular
+                                truncate={descriptionLinesMax}
+                                color={hasMedia ? vars.colors.textPrimary : vars.colors.textSecondary}
+                                textShadow={textShadow}
+                            >
+                                {description}
+                            </Text3>
+                        </div>
+                    )}
+                </Stack>
+                {extra}
+            </div>
+        );
+
+        return (
+            <section
+                {...getPrefixedDataAttributes(dataAttributes, 'CoverHero')}
+                aria-label={ariaLabel}
+                ref={ref}
+                className={classnames(styles.coverHero, {
+                    [styles.centered]: centered,
+                    [styles.hasSideExtra]: sideExtra,
+                    [styles.minHeight]: !noPaddingY,
+                })}
+                style={{
+                    minHeight: aspectRatio && aspectRatio !== 'auto' ? 'auto' : minHeight,
+                    boxSizing: 'border-box',
+                    background,
+                    aspectRatio: aspectRatioToCssString(aspectRatio),
+                    ...applyCssVars({
+                        [mediaStyles.vars.mediaBorderRadius]: '0px',
+                    }),
+                }}
+            >
+                {hasMedia ? <CoverHeroMedia {...mediaProps} /> : null}
+                <ResponsiveLayout variant={hasMedia ? 'inverse' : variant} backgroundColor="none">
+                    <Box paddingY={noPaddingY ? 0 : {desktop: 56, tablet: 56, mobile: 24}}>
+                        <Stack space={24}>
+                            {centered && !sideExtra ? (
+                                mainContent
```

Comment:
When centered in desktop seems that there's no limitation for the title, pretitle and description width.

Same as left aligned the content should take:

Pretitle: 6col (or that 75%)
Title: 8col
Description: 6col (or that 75%)

![Screenshot 2024-06-11 at 18 10 37](https://github.com/Telefonica/mistica-web/assets/44420072/5827b08c-3647-4fe2-8a21-bf06ab45771b)

Reply:
fixed